plugin-docker/che-plugin-docker-runner/src/main/java/org/eclipse/che/plugin/docker/runner/BaseDockerRunner.java

```diff
@@ -319,7 +319,18 @@ protected ApplicationProcess newApplicationProcess(DeploymentSources toDeploy, R
                         unpackedApplication = unpackArchive(application, workDir, applicationFilename + "_unpack");
                     }
                 }
-                hostConfig.setBinds(new String[]{String.format("%s:%s", unpackedApplication.getAbsolutePath(), applicationBindDir)});
+
+                // On Windows binding directory needs to follow URL convention with first / and no colon :
+                // instead of C:\\Users\\user it needs to be /c/Users/user (note as well the lowercase c at first)
```

Comment:
Write in any of your java class c:\users, you'll see that you'll a compilation issue as \u will be interpreted as unicode character. And then you'll understand the issue of why in the comment I added \\